src/test/java/core/basesyntax/service/ReportCreatorServiceTest.java

```diff
@@ -42,7 +42,7 @@ public void createReport_emptyData_notOk() {
     }
 
     @Test
-    public void createReport_NoCsvHeader_notok() {
+    public void createReport_NoCsvHeader_notOk() {
```

Comment:
this test is not expecting any exception and what name should it have?

Reply:
even if it not expecting any exception, it's still wrong case, as report file without header - it's incorrect. Or I got something wrong?